### PR TITLE
fix: updated provider secret key

### PR DESCRIPTION
**Pull request by irediaes**

### CODE REVIEW

**Comment by niazhussain** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **kosswest**
You can assign the PR to them by writing `/assign @kosswest` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=peaqnetwork%2Fpeaq-network-ev-charging-sim-be).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/peaqnetwork/peaq-network-ev-charging-sim-be/blob/dev/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by sfffaaa — APPROVED** [review]

**Comment by niazhussain** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *[sfffaaa](https://github.com/peaqnetwork/peaq-network-ev-charging-sim-be/pull/35#pullrequestreview-941189072 "Approved")*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **kosswest**
You can assign the PR to them by writing `/assign @kosswest` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=peaqnetwork%2Fpeaq-network-ev-charging-sim-be).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/peaqnetwork/peaq-network-ev-charging-sim-be/blob/dev/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by niazhussain** [comment]
:star: PR built and available in a preview **peaqnetwork-peaq-network-ev-charging-sim-be-pr-35** [here](http://peaq-network-ev-charging-sim-be-pr35.ci.peaq.network)

**Pull request merged**
